apm-sniffer/apm-sdk-plugin/mysql-common/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/Constants.java

```diff
@@ -29,4 +40,62 @@
     public static final String SET_CATALOG_INTERCEPTOR = "org.apache.skywalking.apm.plugin.jdbc.mysql.SetCatalogInterceptor";
     public static final String STATEMENT_EXECUTE_METHODS_INTERCEPTOR = "org.apache.skywalking.apm.plugin.jdbc.mysql.StatementExecuteMethodsInterceptor";
     public static final String DRIVER_CONNECT_INTERCEPTOR = "org.apache.skywalking.apm.plugin.jdbc.mysql.DriverConnectInterceptor";
+
+    public static final StringTag SQL_PARAMETERS = new StringTag("db.sql.parameters");
+    public static final String SQL_PARAMETER_PLACEHOLDER = "?";
+    public static final Set<String> PS_SETTERS = new HashSet<String>(Arrays.asList(
```

Comment:
I still have performance concerns for these naming match. You should set a explicit instrumentation for get parameter, rather than using string match.
Instrumentation name match filter happens in startup amd class loading stage.